### ISSUE TITLE
Support contenteditable divs with userEvent.paste

### ISSUE THREAD

**Issue by rbrady-hs** [testing-library/user-event#699]
- `@testing-library/user-event` version: 13.1.9

- Testing Framework and version: jasmine 2.91
  
- DOM Environment: Chrome 91.0.4472.114
  

Relevant code or config

```javascript
describe('contentEditable', () => {
  it('pastes', () => {
    render(<div contentEditable="true" data-testid="input" />);
    userEvent.paste(screen.getByTestId('input'), 'asd');
    expect(screen.getByLabelText('input').value).toEqual('asd');
  });
});
```

What you did:

As above, tried pasting into a div with contenteditable="true"

What happened:

Test fails with the following error message:

```
TypeError: The given DIV element is currently unsupported.
      A PR extending this implementation would be very much welcome at https://github.com/testing-library/user-event
```

Reproduction repository: N/A

Problem description: N/A

Suggested solution: Not too sure what the implementation would look like, but couldn't find a feature request covering this so thought I'd add an issue if that's alright!

**Comment by ph-fritsche** [comment]
Please have a look at #782 .

**Comment by ph-fritsche** [comment]
Resolved in #785
